packages/code-editor/README.md

```diff
@@ -66,6 +66,14 @@ def all_lookml_models(
 }
 ```
 
+### `<CodeDisplay /`
+
+This component extends the `<CodeDisplay />` component and adds a "copy to Clipboard" button
```

Comment:
This component wraps instead of extends? Being an OO purist here

Reply:
Using "wraps" now